It (as well as all previous versions) installs fine locally on Windows 10x64, and in a Windows Sandbox. I reviewed the logs, and there are no details on why it's failing other than the text 
[##[error] Result: Failed](https://dev.azure.com/ms/winget-pkgs/_build/results?buildId=419594&view=logs&j=871b4eed-22de-5cb6-c1c8-82f190de633f&t=523ceef4-ddf7-55a0-c83d-dd0f77b67744&l=55). 
Please direct me where to look so I can diagnose this issue. Thanks.

--- a/manifests/i/ImageMagick/ImageMagick/7.1.0.58/ImageMagick.ImageMagick.locale.en-US.yaml
+++ b/manifests/i/ImageMagick/ImageMagick/7.1.0.58/ImageMagick.ImageMagick.locale.en-US.yaml
@@ -33,7 +33,7 @@ Tags:
 - shear
 # Agreements:
 # ReleaseNotes:
-# ReleaseNotesUrl:
+ReleaseNotesUrl: https://github.com/ImageMagick/Website/blob/main/ChangeLog.md
 # PurchaseUrl:
 # InstallationNotes:
 # Documentations:

